webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsService.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.responders.admin
+
+import zio._
+
+import org.knora.webapi.messages.admin.responder.projectsmessages._
+import org.knora.webapi.messages.admin.responder.usersmessages.UserADM
+import org.knora.webapi.responders.ActorToZioBridge
+
+trait ProjectsService {
+  def getProjectsADMRequest(): Task[ProjectsGetResponseADM]
+  def getSingleProjectADMRequest(identifier: ProjectIdentifierADM): Task[ProjectGetResponseADM]
+  def createProjectADMRequest(
+    payload: ProjectCreatePayloadADM,
+    requestingUser: UserADM
+  ): Task[ProjectOperationResponseADM]
```

Comment:
Not entirely sure what the reasons for renaming were. IMHO, the signature of the methods suggest that this service should indeed be named `Rest…` service.

Indicators are: 
* The return values are all `*ResponseADM` instances. 
* The`createProjectADMRequest` takes a user as an argument

IMHO, we should aim at separation of concerns and reflect that in the names of our new components.
I.e. `*Route`s handle mostly presentation logic; `Rest*Service`s handle aggregation of business logic calls as well as permissions mapping; `*Service`s  contain business logic; `*Repo`s handle persistence.

Wdyt?

Reply:
This happened during the mob session, when we were all not sure, what the deal was with the `Rest*` name.

I fully agree that we should aim at separation of concerns and should reflect that in our naming. But for conventions to be meaningful, we should all know them. ;-)  
Also, as a personal matter of taste, I don't like `Rest*Service`. I can absolutely be happy about a prefix or a suffix based naming convention, but I don't like circumfix based naming conventions.

In any case, this sounds like a perfect case for a RFC, right?

But for now, this shouldn't block us from merging. Shall I revert the name change, or leave it as is for now?

Reply:
`Rest*Service` is my suggestion and it is what we have used so far and what I am currently using in my other PR.